server/service/integration_enterprise_test.go

```diff
@@ -51,8 +53,9 @@ func (s *integrationEnterpriseTestSuite) SetupSuite() {
 		License: &fleet.LicenseInfo{
 			Tier: fleet.TierPremium,
 		},
-		Pool: s.redisPool,
-		Lq:   s.lq,
+		Pool:   s.redisPool,
+		Lq:     s.lq,
+		Logger: log.NewLogfmtLogger(os.Stdout),
```

Comment:
What will this code do?
stream stuff to stdout?

Reply:
If so, do we want that when running the test?

Reply:
Yes. (This is a test file, so it's ok if we log stuff to stdout for troubleshooting.)

Reply:
`go test` will only print the tests stdout if using `-v`.